cli/src/commands/rebase.rs

```diff
@@ -134,18 +134,18 @@ use crate::ui::Ui;
 /// commit. This is true in general; it is not specific to this command.
 #[derive(clap::Args, Clone, Debug)]
 #[command(verbatim_doc_comment)]
-#[command(group(ArgGroup::new("to_rebase").args(&["branch", "source", "revisions"])))]
+#[command(group(ArgGroup::new("to_rebase").args(&["bookmark", "source", "revisions"])))]
 #[command(group(ArgGroup::new("target").args(&["destination", "insert_after", "insert_before"]).multiple(true).required(true)))]
 pub(crate) struct RebaseArgs {
-    /// Rebase the whole branch relative to destination's ancestors (can be
+    /// Rebase the whole bookmark relative to destination's ancestors (can be
     /// repeated)
     ///
     /// `jj rebase -b=br -d=dst` is equivalent to `jj rebase '-s=roots(dst..br)'
     /// -d=dst`.
     ///
     /// If none of `-b`, `-s`, or `-r` is provided, then the default is `-b @`.
     #[arg(long, short)]
-    branch: Vec<RevisionArg>,
+    bookmark: Vec<RevisionArg>,
```

Comment:
Does this argument refer to a branch in the graph itself? It might make more sense to leave it as "branch" in that case, since it doesn't have to correspond to a bookmark.

Reply:
Good find!

Reply:
reverted